Source/santad/SNTApplication.m

```diff
@@ -68,7 +69,13 @@ - (instancetype)init {
       return nil;
     }
 
-    _eventLog = [[SNTEventLog alloc] init];
+    // Choose an event logger.
+    SNTConfigurator *configurator = [SNTConfigurator configurator];
+    if ([[configurator eventLogType] isEqualToString:@"Syslog"]) {
```

Comment:
Also case-insensitive.

Alternatively, have the getters in SNTConfigurator parse these strings and convert to an enum so you can use a switch here.

Reply:
Done, moved to enum.